### ISSUE TITLE
Redesign media and flux logs?

### ISSUE THREAD

**Issue by mquintin** [segrelab/comets#7]
Issue submitted because they take forever to load directly into MATLAB tables

(This issue copied from the master list at https://sites.google.com/a/bu.edu/segre-lab/projects/comets/comets-issues . It may have been resolved and left unmarked as such)

**Comment by mquintin** [comment]
Would it be feasible to alter media logs by being able to designate only certain metabolites to be logged? I'm not sure what the common use case is, but in my particular case I'm only interested in 5/200

**Comment by djbajic** [comment]
Hey Mike, Im going to look into it, Im also not very happy with the current format of these logs..

**Comment by djbajic** [comment]
I did some changes in the branch format_outputs, basically simplifying both media and flux logs

- The media logs now write, for each timepoint and cell, only the concentration of those metabolites that is !=0. This reduces **a lot** the size of the file. 
- The flux log, I did not change it much, I just removed the weird brackets around numbers. 

I wanted to suggest that we remove all the different formating options for the logs in comets, and agree to a unique, as simple as possible, format for all the output files. This would simplify the code, and any analysis software can take a csv table for instance. I feel this is important so I want to know your opinions before merging anything or going into more deep modifications. 

In the meantime, I hope  this quick fix saves you some time Mike.
